docs/index.md

```diff
@@ -0,0 +1,26 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "dexidp Provider"
+subcategory: ""
+description: |-
+  Provision Dex resources via the Dex gRPC API.
+---
+
+# dexidp Provider
+
+Provision Dex resources via the Dex gRPC API.
+
+## Example Usage
+
```

Comment:
Make sure that Dex server is running with grpc api reachable via 127.0.0.1:5557

Reply:
This is generated.

Reply:
Updated it in the example by adding a comment to terraform